libs/components/src/lib/search-chip/index.tsx

```diff
@@ -0,0 +1,73 @@
+import styles from './search.module.css';
+import { qtMerge } from '@deriv/quill-design';
+import { StandaloneSearchRegularIcon } from '@deriv/quill-icons';
+import { useEffect, useRef, useState } from 'react';
+
+export type sizeVariant = 'sm' | 'md';
+
+export type InputProps = {
+  placeholder?: string;
+  value?: string;
+  onChange: (e: string, i?: React.FormEvent<HTMLInputElement>) => void;
+  className?: string;
+  inputClassName?: string;
+  size?: sizeVariant;
+};
+
+const sizeVariantClass: { [key in sizeVariant]: string } = {
+  sm: 'px-general-sm py-general-xs',
+  md: 'p-general-md',
+};
+
+export const SearchChip = ({
```

Comment:
can we use `SearchInput` from quill-design please?

Reply:
removed this file as we are no more using search chip